build.gradle

```diff
@@ -40,6 +40,10 @@ subprojects { project ->
     }
 }
 
+task wrapper(type: Wrapper) {
```

Comment:
Lets leave this out. Its easier to just invoke the version on the commandline usually